### PR TITLE
Practice part2 completed

### PR DESCRIPTION
**Pull request by MarynaKushniruk**

### CODE REVIEW

File: src/main/java/com/example/bookstore/repository/cartitem/CartItemRepository.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+package com.example.bookstore.repository.cartitem;
+
+import com.example.bookstore.model.CartItem;
+import java.util.Set;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.query.Param;
+
+public interface CartItemRepository extends JpaRepository<CartItem, Long> {
+    @Query("SELECT c FROM CartItem c WHERE c.shoppingCart.id =:shoppingCartId")
```
Comment (olekskov) [inline src/main/java/com/example/bookstore/repository/cartitem/CartItemRepository.java]: such simple queries should be handled by proper method naming, please recheck work without them and fix just naming method if required

File: src/main/java/com/example/bookstore/repository/shoppingcart/ShoppingCartRepository.java [hunk 2]
```diff
@@ -0,0 +1,11 @@
+package com.example.bookstore.repository.shoppingcart;
+
+import com.example.bookstore.model.ShoppingCart;
+import java.util.Optional;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.jpa.repository.Query;
+
+public interface ShoppingCartRepository extends JpaRepository<ShoppingCart, Long> {
+    @Query("SELECT sc FROM ShoppingCart sc WHERE sc.user.id =:userId")
```
Comment (olekskov) [inline src/main/java/com/example/bookstore/repository/shoppingcart/ShoppingCartRepository.java]: same here

File: src/main/java/com/example/bookstore/repository/cartitem/CartItemRepository.java [hunk 3]
```diff
@@ -0,0 +1,10 @@
+package com.example.bookstore.repository.cartitem;
+
+import com.example.bookstore.model.CartItem;
+import java.util.Set;
+import org.springframework.data.jpa.repository.JpaRepository;
+import org.springframework.data.repository.query.Param;
+
+public interface CartItemRepository extends JpaRepository<CartItem, Long> {
+    Set<CartItem> findCartItemsByShoppingCartId(@Param("shoppingCartId") Long shoppingCartId);
```
Comment (Ivan95kos) [inline src/main/java/com/example/bookstore/repository/cartitem/CartItemRepository.java:9]:
```suggestion
    Set<CartItem> findCartItemsByShoppingCartId(Long shoppingCartId);
```

**Review by Ivan95kos — APPROVED** [review]
Great job, with minor comments

**Pull request merged**